sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -96,8 +96,8 @@ def order_lro_results(doc_id_order, combined):
 def prepare_result(func):
     def choose_wrapper(*args, **kwargs):
         def wrapper(
-            response, obj, response_headers, ordering_function
```

Comment:
`response_headers` was never used in any of these response handler functions (and is an empty dict anyway), but we passed it around because the callbacks passed it. Refactored the code to remove it where we can in this file since we're passing the `bespoke` kwarg now.